config.txt.sample

```diff
@@ -1,22 +1,68 @@
+# Database connection configuration section
+# Use this section to specify connection configuration for PostgreSQL
+# database. The values in this section can also be supplied by
+# environment variables with keys matching those in the block below.
 [DB]
-PGHOST=localhost
-PGUSER=postgres
-PGPASSWORD=admin
-PGDATABASE=raw
-PGPORT=5432
+PGHOST=postgresql.example.net
+PGUSER=rawdatadba
+PGPASSWORD=_superSecureS3cretString!_
+PGDATABASE=rawdata
+;PGPORT=5432
 
+# OAuth2 app client configuration section
+# When you create an OAuth2 application on OpenStreetMap portal, place
+# the returned values here. Currently raw-data API only uses read_prefs
+# for scope. Callback / Redirect URI should be set when the OAuth2 app
+# is created on OSM portal. App secret key is a high-entropy string. 
 [OAUTH]
-OSM_CLIENT_ID= your client id
-OSM_CLIENT_SECRET= your client secret
-OSM_URL=https://www.openstreetmap.org
-OSM_PERMISSION_SCOPE=read_prefs
-LOGIN_REDIRECT_URI=http://127.0.0.1:8000/v1/auth/callback/
-APP_SECRET_KEY=
+OSM_CLIENT_ID=0yhbzr7-e0F-DGgyxXOyq2Y4d_sYLflMzVvLIfRtxHg
+OSM_CLIENT_SECRET= KI59u4PLCQFAd8Vo2oewaHNmlf2EfTcfGgwWd2VCtk8 
+;OSM_URL=https://www.openstreetmap.org
```

Comment:
Is ';'  character before OSM_URL  intentional ?

Reply:
Yes, INI files allow for comments that begin with either `#` or `;` . both are valid start characters for comment lines. I have also verified that configparser understands this.